docs/spec.md

```diff
@@ -1862,13 +1872,14 @@ Produces an `output` tensor from a constant `value`.
 
 #### Outputs
 
-| Name     | Type   | Constraints |
-|----------|--------|-------------|
-| `output` | tensor | (C1)        |
+| Name     | Type                                  | Constraints |
+|----------|---------------------------------------|-------------|
+| `output` | tensor or per-tensor quantized tensor | (C1)        |
 
 #### Constraints
 
-* (C1) `type(value) = type(output)`.
+* (C1) `type(value) = is_quantized(output) ? storage_type(output) :
+       type(output)`.
```

Comment:
I'm not sure I follow why this change is needed. Why wouldn't `type(value)` be equal to `type(output)` for the quantized case?

Reply:
You are right. I confused `value` with individual tensor elements. updated.